features/anti-link.js

```diff
@@ -19,12 +21,31 @@ module.exports = (client) => {
           .setFooter('Message sent by the Heptagram Bot', `${cdn.sqlogo}`);
 
         channel.send({ embeds: [nolinkembed] });
+
       });
+
+      //To warn the user when a link is sent
+      try{
+        const arguments = [`${message.author}`, `Links not allowed`]
```

Comment:
`Parsing error: Binding 'arguments' in strict mode.` 
and
`Shadowing of global property 'arguments'.`

Reply:
Changed the `arguments` to `argsAssign`, which should hopefully not cause any error.
Also, if I am not submitting anything properly through git, then pls tell, because I am still learning git